src/GafferScene/FreezeTransform.cpp

```diff
@@ -87,21 +87,37 @@ void FreezeTransform::affects( const Gaffer::Plug *input, AffectedPlugsContainer
 {
 	FilteredSceneProcessor::affects( input, outputs );
 
-	if( input == inPlug()->transformPlug() )
+	if(
+		input == inPlug()->transformPlug() ||
+		input == outPlug()->transformPlug()
```

Comment:
I don't get why this would depend on the outPlug() transform?

Reply:
The short answer is "look at the compute" - if `affects()` doesn't match then something is wrong. The slightly longer answer is that it's perfectly permissible for a compute to depend on another output of the same node (that's how internal plugs are useful at all), and in this case the easiest way to figure out what transform to freeze into the vertices is to compare the input and output transforms.